client/__tests__/e2e/e2e.test.ts

```diff
@@ -688,7 +688,7 @@ for (const option of options) {
 
           await setup(option, page);
 
-          waitUntilNoSkeletonDetected(page);
+          await waitUntilNoSkeletonDetected(page);
```

Comment:
`setup()` explicitly calls `waitUntilNoSkeletonDetected()`

Reply:
nice call!

Reply:
True - I've reverted that last change